apps/api/src/resources/swagger.yaml

```diff
@@ -634,14 +634,14 @@ paths:
     delete:
```

Comment:
We should create a follow up ticket for adding Auth on Swagger?

Reply:
Oh interesting, good point. This should likely interact with your logged in session, but needs further investigation.

Reply:
Discussed offline - Auth gets applied on Swagger, but userId fields are not functional.
Implementing auth/swagger should be a follow up ticket.